### DISCUSSION TITLE
Get points of a subtask

### DISCUSSION THREAD

**Discussion by zentox** [tudace/tuda_latex_templates#486] [General LaTeX Questions] [answered]
I wanted to know if there is a similar command to `\getPoints{\thetask}` which works for subtasks and if there is also a `mapPoints` command for subtasks.

**Comment by TeXhackse** [comment, marked as answer]
Sorry … somehow I did not get any message about this. Yes now there is and it's now documented as well. See tudaexercise demo for an example.
